src/App.tsx

```diff
@@ -35,10 +35,12 @@ const StyledApp = styled.div`
 const StyledPageBody = styled.div`
   display: flex;
   flex-direction: column;
-  align-items: center;
+  @media (min-width: 600px) {
```

Comment:
vi burde lagt terskelverdiene for media-query i konstanter. De brukes flere steder og vi kommer sikkert til å justere tallene.

Reply:
helt enig 👍